eclipse/sw360
# fix(Rest-API): Small fix around ProjectClearingState during create and update project

1.While creating project from rest end point, "Project clearing state" should set to "open" by default if not set explicitly.
2. Only users with Admin rights should be allowed to edit a closed project.Small fix was needed from  for this feature to work from rest end point.

Steps to test:
*  Scenario 1:
   *  Create a project from rest end point without setting the "Project clearing state" field.
   *  Project clearing state value should set to "Open" by default.
* Scenario 2:
  *  Open a project and mark the "Project clearing state" as "closed".
  *  Now try to update the same project from rest end point. If the user does not have admin rights then the user should not be allowed to edit the project.
  *  Users with admin rights  should still be allowed to edit the closed project.
  *  Change the "Project clearing state" as "open" .Now all the users who has access to modify the projects should be allowed to edit the project from rest end point.

Signed-off-by: Nutan Das <nutan.das@siemens.com>
